pkg/client/app.go

```diff
@@ -96,6 +107,12 @@ func ToAppUpdate(ctx context.Context, c Client, name string, opts *AppUpdateOpti
 	}
 
 	if opts.Image != "" {
+		img, err := c.ImageGet(ctx, opts.Image)
```

Comment:
Just curious, why not also use the `GetImageRef` function here?

Reply:
Because I forgot to apply it here :grimacing: